src/components/MainRouter.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { LoggedInPrompt } from "../pages/LoggedInPrompt";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Navigation from "./Navigation";
+import Language from "./Language";
+import Quiz from "./Quiz";
+import Help from "../pages/Help";
+import Settings from "../pages/Settings";
+
+export const MainRouter = () => {
+  return (
+    <Router>
+      <Navigation />
+
+      <Switch>
+        <Route path="/index.html">
```

Comment:
Did you try with this? Not that it matters much, just a bit more flexible
```suggestion
        <Route path="/" exact>
```

Reply:
Yep! Unfortunately, this is one of the many quirks when working with an extension (took us a while to figure out the correct path 😅)